docs/cardinal/game/configuration.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuration'
+description: 'Configuring the Cardinal game shard'
+---
+
+Each service in the World Engine stack reads its configuration values from environment variables. Using the [World CLI](https://github.com/Argus-Labs/world-cli) makes configuring your project even easier via its `world.toml` file, where all World Engine configuration values are in one place.
+
+Below are the key value pairs that can be set in either the `world.toml` file or directly in environment variables.
+
+
+## Cardinal `[cardinal]`
+| Name                         | Description                                                                                                                                                                                                          | Required for Production |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| CARDINAL_NAMESPACE           | The name of your cardinal game shard. If you are running multiple game shards in a single World Engine instance, this MUST be unique for each game shard. Must only be alphanumeric characters and hyphens (-).      | YES                     |
+| CARDINAL_MODE                | The cardinal mode determines how your game shard runs. Must be either "development" or "production". In development, Cardinal will not submit any transactions to the base shard, and security concerns are lowered. | N/A                     |
+| REDIS_ADDRESS                | The address of the redis instance.                                                                                                                                                                                   | YES                     |
+| REDIS_PASSWORD               | The password of your redis instance.                                                                                                                                                                                 | YES                     |
+| CARDINAL_LOG_LEVEL           | The level of logging Cardinal should output. See [Leveled Logging](https://github.com/rs/zerolog?tab=readme-ov-file#leveled-logging) for more details.                                                               | NO                      |
```

Comment:
Provide list of valid log level strings

Reply:
fixed [nits](https://github.com/Argus-Labs/world-engine/pull/736/commits/ec1345c83691c5e1ea44e62550f72af71b52e2d1)